roles/mount_discovery_iso_for_pxe/defaults/main.yml

```diff
@@ -1,17 +1,23 @@
 ---
 # defaults file for generate_discovery_iso
 tftp_root: "{{ hostvars['tftp_host']['tftp_directory'] | default('/var/lib/tftpboot/') }}"
-mount_directory: "/mnt/images"
+mount_directory: "/mnt/iso/images"
```

Comment:
Maybe you can call it `mount_images_directory` and create another variable like `mount_efiboot_directory` so you can use it instead of using the full path in your playbook